File: ce/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples21.vb
' <snippetlinqexamples21>


 Using svcContext As New ServiceContext(_serviceProxy)
  Dim query_gt2 = From c In svcContext.ContactSet _
                  Where c.CreditLimit.Value > 20000 _
                  Select New With {Key c.FirstName,
                                   Key c.LastName,
                                   Key c.Address1_City}
  For Each c In query_gt2
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName _
                     &amp; " " &amp; c.Address1_City)
  Next c
 End Using

' </snippetlinqexamples21>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/linqexamples21.cs
// <snippetlinqexamples21>


 using (ServiceContext svcContext = new ServiceContext(_serviceProxy))
 {
  var query_gt2 = from c in svcContext.ContactSet
                  where c.CreditLimit.Value > 20000
                  select new
                  {
                   c.FirstName,
                   c.LastName,
                   c.Address1_City
                  };
  foreach (var c in query_gt2)
  {
   System.Console.WriteLine(c.FirstName + " " +
    c.LastName + " " + c.Address1_City);
  }
 }

// </snippetlinqexamples21>